ui/app/components/app/account-menu/account-menu.component.js

```diff
@@ -193,6 +193,12 @@ export default class AccountMenu extends Component {
 
   renderRemoveAccount (keyring, identity) {
     const { t } = this.context
+
+    // Sometimes keyrings aren't loaded yet
```

Comment:
Without it, in the `account-menu.test.js`, the component will render with 
>TypeError: Cannot destructure property `type` of 'undefined' or 'null'.

which refers to `state.metamask.keyrings`

After this change the component will render correctly with the props. This was the only way to get it to render and do basic tests on it. I just copied that lines from `removeAccount()`, https://github.com/MetaMask/metamask-extension/blob/7b730ebc472a86c8e9b6f12f8136b20131fed544/ui/app/components/app/account-menu/account-menu.component.js#L229-L232 since it seems that it doesn't initially load.